src/lib/core/rtl/directionality.ts

```diff
@@ -0,0 +1,18 @@
+import {EventEmitter, Injectable} from '@angular/core';
+import {LayoutDirection} from './dir';
+
+/**
+ * A provider to get the direction outside of angular on the `html` or `body` elements
```

Comment:
"A provider" isn't really accurate (it's just a class that's _provided_). It can just be something like
```
The directionality (LTR / RTL) context for the application (or a subtree of it). 
Exposes the current direction and a stream of direction changes.
```